SU2_CFD/include/drivers/CDriverBase.hpp

```diff
@@ -312,7 +312,7 @@ class CDriverBase {
    * \param[in] iVertex - Marker vertex index.
    * \return Marker vertex.
    */
-  unsigned long GetMarkerNode(unsigned short iMarker, unsigned long iVertex) const;
+  unsigned long GetMarkerNodeIndex(unsigned short iMarker, unsigned long iVertex) const;
```

Comment:
Yes this one for example.
I'm using GetNode<Property>, GetElement<Property>, etc. so adding Index doesn't really clarify much (I wanted to use Point instead of Node, but tried to respect some of the original author's personal preferences).
If you want to make GetNodeDomain clearer then call it GetNodeIsDomain to keep the convention.

You can remove underscores as you please, our use of `Worm_Case` is an eyesore. Even worse is the `What_theHeckcase`

Please try to not mix everything in a single PR, this title makes me uneasy, tackle the modernization, the proper case styles, and proposed changes to the python interface separately.

Reply:
I see the logic now, the index is dropped as one can suggest from the return type `unsigned int` and is x in/at/on y  
from return type bool. 

Thanks.